lib/base/index.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+import {keybaseExec} from '../utils'
+import {withStore, actionInit} from '../store'
+import type {Store} from '../store/types'
+import type {InitOptions, DeviceUsernamePair, UsernameAndDevice} from '../types.js'
+
+/**
+ * @ignore
+ * _getCurrentUsernameAndDevicename returns { username, devicename } from keybase status -j
+ */
+const _getCurrentUsernameAndDevicename = async (): Promise<UsernameAndDevice> => {
```

Comment:
Since most (all?) underscore methods in Bot are now pure functions 😍 can we get rid of the underscores? No need for hacky JS object-oriented stuff!

Reply:
Do underscored functions do something in classes? I always thought it was a convention for denoting private methods desire JavaScript not having private methods.

Reply:
Oh yeah, the underscores are just a naming convention. But since `_getCurrentUsernameAndDevicename` isn't a private method anymore, not really sure it makes sense.